packages/protocol/contracts/verifiers/compose/ZkVMVerifier.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "./ComposeVerifier.sol";
+
+/// @title ZkVMVerifier
+/// @notice This contract is a verifier for the Mainnet ZkVM that composes RiscZero and SP1
+/// Verifiers.
+/// @custom:security-contact security@taiko.xyz
+contract ZkVMVerifier is ComposeVerifier {
```

Comment:
Just think maybe also impl TEEVerifier in this PR since the framework code is exactly the same.